src/blockchain/ain/jellyfish/raw-tx-util.ts

```diff
@@ -58,6 +59,35 @@ export class RawTxUtil {
     return [prevouts, rawScriptHex];
   }
 
+  static parseUnspentUntilAmount(unspent: AddressUnspent[], amount: BigNumber): [Prevout[], BigNumber, string] {
+    let rawScriptHex = '';
+    const prevouts = unspent.map((item): Prevout => {
+      rawScriptHex = item.script.hex;
```

Comment:
Why is this overriden for every unspent?

Reply:
shouldn't matter, as every unspent should have the same `script.hex`. This is needed to correctly reconstruct `Prevout` object/s on cold-wallet